projects/extension/src/background/types.ts

```diff
@@ -3,6 +3,13 @@ import { AppMediator } from './AppMediator';
 import EventEmitter from 'eventemitter3';
 import StrictEventEmitter from 'strict-event-emitter-types';
 
+export interface HealthResponse {
```

Comment:
We should just use the `SmoldotHealth` type from @substrate/smoldot-light rather than add a separate new type of our own.  The shape is identical